.github/workflows/release_prep.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+set -x -e -u -o pipefail
+
+if [[ $# -ne 1 ]]; then
+    >&2 echo "Usage: ${0} <version tag>"
+    exit 1
+fi
+
+NAME="protovalidate"
+TAG="${1}"
+PREFIX="${NAME}-${TAG:1}"
+ARCHIVE="${PREFIX}.tar.gz"
+
+# Update MODULE.bazel version
+>&2 echo "# Update MODULE.bazel version to ${TAG:1}"
```

Comment:
why are we redirecting all output to stderr?

Reply:
Not any specific reason, I just tend to use `stderr` for logging and `stdout` for results, so I tend to do this without really thinking much.